Creating Conversation Quickstart for Go (#1128)
dapr/quickstarts: conversation/go/sdk/README.md

> # Dapr Conversation

Reviewer: ```suggestion
# Dapr Conversation API (Go SDK)
```

> In this quickstart, you'll send an input to a Large Language Model (LLM) using Dapr's Conversation API. This API is responsible for providing one consistent API entry point to talk to underlying LLM providers.

Reviewer: ```suggestion
In this quickstart, you'll send an input to a mock Large Language Model (LLM) using Dapr's Conversation API. This API is responsible for providing one consistent API entry point to talk to underlying LLM providers.
```

Visit [this](https://v1-15.docs.dapr.io/developing-applications/building-blocks/conversation/conversation-overview/) link for more information about Dapr and the Conversation API.

> > **Note:** This example leverages HTTP `requests` only.  If you are looking for the example using the HTTP API [click here](../http/).

Reviewer: This is wrong and needs to be removed. It uses SDK

This quickstart includes one app:

- `conversation.go`, responsible for sending and input to the underlying LLM and retrieving an output.

## Run the app with the template file

This section shows how to run the application using the [multi-app run template files](https://docs.dapr.io/developing-applications/local-development/multi-app-dapr-run/multi-app-overview/) with `dapr run -f .`.  

> This example uses the Default LLM Component provided by Dapr for testing purposes. Here are other [supported Conversation components](https://v1-15.docs.dapr.io/reference/components-reference/supported-conversation/).

Reviewer: Can you mention that echo just spits out what is put in? More details here plz

Reviewer: Also since we are going to use the echo component in Conversation, can we link to any other Go examples that are more complex using anthropic or something? maybe in go sdk repo

Author: I don't believe these examples exist. @mikeee do you know?

Open a new terminal window and run the multi app run template:

<!-- STEP
name: Run multi app run template
expected_stdout_lines:
  - '== APP - conversation == Input sent: What is dapr?'
  - '== APP - conversation == Output response: What is dapr?'
expected_stderr_lines:
output_match_mode: substring
match_order: none
background: false
sleep: 15
timeout_seconds: 30
-->

```bash
dapr run -f .
```

The terminal console output should look similar to this, where:

> - The app send an input `What is dapr?` to the `echo` LLM.

Reviewer: ```suggestion
- The app sends an input `What is dapr?` to the `echo` Component mock LLM.
```

> - The LLM echoes `What is dapr?`.

Reviewer: ```suggestion
- The mock LLM echoes `What is dapr?`.
```
